app/assets/javascripts/geoblacklight/templates/index_map_info.hbs

```diff
@@ -14,6 +14,15 @@
         <img src="{{thumbnailUrl}}">
       {{/if}}
     {{/if}}
+    {{#if thumbUrl}}
+      {{#if websiteUrl}}
+        <a href="{{websiteUrl}}">
+          <img src="{{thumbUrl}}" class="img-responsive">
```

Comment:
Let's change the `img-responsive` CSS class names in this whole file to `img-fluid`.

Bootstrap v4 migrated to `img-fluid` from `img-responsive` and we just never fixed that in GBL: https://getbootstrap.com/docs/4.6/migration/#images

- [ ] thumbUrl class
- [ ] thumbnailUrl (used just above here)

Reply:
done!